string arm_config_req
---
string arm_config_resp
